### PR TITLE
CONJ-4695 - Buildpack is updated to pull in Conjur version / ssl cert

### PR DESCRIPTION
**Pull request by izgeri**
This PR updates the Conjur Buildpack to pull in the Conjur version and SSL cert from `VCAP_SERVICES`. This update enables the Buildpack to be used with the enterprise version of Conjur.

[Jenkins build](https://jenkins.conjur.net/job/cyberark--cloudfoundry-conjur-buildpack/job/CONJ-4695-buildpack-supports-v4-sb/1/)
[Jira story](https://ca-il-jira.il.cyber-ark.com:8443/browse/CONJ-4695)

### CODE REVIEW

File: lib/0001_retrieve-secrets.sh [hunk 1]
```diff
@@ -68,12 +72,14 @@ def generate_vcap_creds_hash
     creds.has_key? key.to_s
   end
 
+  @creds_map.reject! { |key, _| creds[key.to_s].empty? }
```
Comment (jtuttle) [inline lib/0001_retrieve-secrets.sh]: Is this line just intended to remove the cert from the return value if it's an empty string? Can you move it below the `valid_creds` conditional block that follows? It seems out-of-place here.

File: lib/0001_retrieve-secrets.sh [hunk 2]
```diff
@@ -68,12 +72,14 @@ def generate_vcap_creds_hash
...
+
   if !valid_creds
     STDERR.puts("malformed '#{@service_label}' credentials in VCAP_SERVICES")
     exit(1)
   end
 
-  Hash[ @creds_map.map { |key, env_var_name| [ env_var_name, creds[key.to_s] ] } ]
+  Hash[ @creds_map.map { |key, env_var_name| [ env_var_name, creds[key.to_s].to_s ] } ]
```
Comment (jtuttle) [inline lib/0001_retrieve-secrets.sh]: What are the implications of this `to_s`? It's just converting the `CONJUR_VERSION`? Does this end up mattering somewhere?

Comment (izgeri) [reply lib/0001_retrieve-secrets.sh]: it's converting the CONJUR_VERSION to a string - it throws an error without it (I didn't identify exactly where), and doesn't affect anything downstream with it

File: lib/0001_retrieve-secrets.sh (hunk 1 above)
Comment (izgeri) [reply lib/0001_retrieve-secrets.sh]: yeah, I thought I already moved it. it should be set now.

**Pull request merged**